### PR TITLE
Add option to suppress submodules (#366)

### PR DESCRIPTION
**Pull request by mattmc3**
Add the `submodule:off` option to suppress repos submodules in the bundle. The default is "on", and the docs have been updated.

See issue #366.

### CODE REVIEW

File: project/git.go [hunk 1]
```diff
@@ -144,6 +155,14 @@ func (g gitProject) Update() error {
 	return nil
 }
 
+func recurseSubmodules(str string) bool {
+	switch strings.ToLower(str) {
+	case "0", "f", "false", "off", "no":
```
Comment (caarlos0) [inline project/git.go:160]: I think we could make this simpler and allow only `off` (or `false`) 🤔 wdyt?

If we decide to support all these options, we should probably add tests for them as well.

**Comment by caarlos0** [comment]
closes #366

**Comment by caarlos0** [comment]
Thanks for the PR @mattmc3 , made just a small comment above!

Cheers!

File: project/git.go (hunk 1 above)
Comment (mattmc3) [reply project/git.go:160]: LOL, I agree. I basically copied [strconv.ParseBool](https://golang.org/src/strconv/atob.go?s=351:391#L1)'s implementation and added "no" and some case insensitivity, but we could pare this way down to not be so permissive. It also eats the error and assumes true if an invalid value is provided. That might not be what we want, but since disabling submodules is an edge case I didn't figure it was a big deal. I can iterate on this some more and add a test.

Comment (caarlos0) [reply project/git.go:160]: I think we can change this to just parse bool with case insensitivity... so its simpler :)

**Comment by caarlos0** [comment]
This repo has been deprecated. Please check the notice in the README.

Thanks
